packages/cli/src/controllers/communityPackages.controller.ts

```diff
@@ -174,15 +169,15 @@ export class NodesController {
 			}
 		}
 
-		let hydratedPackages = this.communityPackageService.matchPackagesWithUpdates(
+		let hydratedPackages = this.communityPackagesService.matchPackagesWithUpdates(
 			installedPackages,
 			pendingUpdates,
 		);
 
 		try {
-			const missingPackages = this.config.get('nodes.packagesMissing') as string | undefined;
+			const missingPackages = config.getEnv('nodes.packagesMissing') as string | undefined;
```

Comment:
```suggestion
			const missingPackages = config.getEnv('nodes.packagesMissing');
```

Unless this is intentional and `undefined` can somehow end up here?

Reply:
moved `nodes.packagesMissing` into a private property on the service.